api/src/main/java/org/openmrs/module/fhir/api/util/FHIRMedicationUtil.java

```diff
@@ -0,0 +1,140 @@
+package org.openmrs.module.fhir.api.util;
+
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Coding;
+import org.hl7.fhir.dstu3.model.Medication;
+import org.hl7.fhir.exceptions.FHIRException;
+import org.openmrs.Concept;
+import org.openmrs.ConceptMap;
+import org.openmrs.ConceptName;
+import org.openmrs.Drug;
+import org.openmrs.DrugIngredient;
+import org.openmrs.DrugReferenceMap;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public final class FHIRMedicationUtil {
+
+    // needed because name is required
+    private static final String DRUG_NAME_PLACEHOLDER = "drugName";
+
+    public static Medication generateMedication(Drug drug) {
+        Medication medication = new Medication();
+
+        medication.setId(drug.getUuid());
+
+//        medication.setForm(generateForm(drug.getDosageForm()));
```

Comment:
why this code is commented out?

Reply:
It was my first approach to set form. Then I found another and correct way to do so and forgot to remove this line. Thanks for the notice.